atom/browser/api/atom_api_net.cc

```diff
@@ -22,12 +30,48 @@ v8::Local<v8::Value> Net::Create(v8::Isolate* isolate) {
   return mate::CreateHandle(isolate, new Net(isolate)).ToV8();
 }
 
+void Net::StartLogging(mate::Arguments* args) {
+  if (net_log_) {
```

Comment:
Is there a case where `net_log_` would not be here, and if so should we be throwing an error for that case (`args->ThrowError`).  Not sure the case where `net_log_` won't be a thing

Reply:
💯 I added the checks when I first thought the `NetLog` is optionally instantiated in Electron based on `--log-net-log`. Let me have them removed 😸